apps/site/assets/css/_grid-button.scss

```diff
@@ -9,6 +9,54 @@
   margin: $base-spacing * -.25;
 }
 
+.c-grid-buttons-right {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+  align-items: flex-end;
+  margin-top: 1rem;
+  @include media-breakpoint-up(sm) {
+    border-bottom: 2px solid $gray-lightest;
+    padding-bottom: 10px;
+  }
+}
+
+.c-location-buttons-container {
+  display: flex;
+  flex-grow: 2;
+  @include media-breakpoint-down(xs) {
+    flex-wrap: wrap;
+  }
+}
+
+.c-location-header {
+  flex-grow: 2;
+  @include media-breakpoint-down(sm) {
+    flex-basis: 100%;
+  }
+}
+
+.c-location-button {
+  flex-grow: 1;
+  height: 50px;
+  width: 100px;
+  font-weight: $font-weight-medium;
+  display: block;
+  flex-shrink: 0;
+  margin-bottom: $space-6;
+  margin-top: $space-6;
+  padding: 0 $base-spacing * .25;
```

Comment:
nitpick: Let's remove this padding for the smallest breakpoint. that way the button width will be aligned with the header 
<img width="45" alt="image" src="https://user-images.githubusercontent.com/2136286/169856429-fb88dede-41c0-4c2b-9e0a-c128083000a3.png">